ethereum/api/build.gradle

```diff
@@ -27,6 +27,12 @@ jar {
   }
 }
 
+configurations { testArtifacts }
+task testJar (type: Jar) {
```

Comment:
why does the test jar need to be published? Can this not be done with the testSupport?

Reply:
gone. It wasn't required.